src/SDKConnector.php

```diff
@@ -250,13 +250,23 @@ private function buildCredentials(KeyInterface $key): CredentialsInterface {
    */
   protected function userAgentPrefix(): string {
     if (NULL === self::$userAgentPrefix) {
-      $module_info = $this->infoParser->parse($this->moduleHandler->getModule('apigee_edge')->getPathname());
-      if (!isset($module_info['version'])) {
-        $module_info['version'] = '8.x-1.x-dev';
+
+      $edge_module_info = $this->infoParser->parse($this->moduleHandler->getModule('apigee_edge')->getPathname());
+      if (!isset($edge_module_info['version'])) {
+        $edge_module_info['version'] = '8.x-1.x-dev';
+      }
+      $edge_module_info = $edge_module_info['name'] . '/' . $edge_module_info['version'];
+
+      $m10_module_info = '';
+      if ($this->moduleHandler->moduleExists('apigee_m10n')) {
```

Comment:
This should be added by the apigee_m10n module to the user agent string, the dependency must not know about every of its dependants.

Reply:
@shishir-intelli  @phdhiren What do you think about my comment?

Reply:
Let's implement via custom hook and let apigee_m10n module extend this.

Reply:
Thanks @mxr576,
That makes sense to me.